runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/ClueScrollWorldMapPoint.java

```diff
@@ -69,4 +79,41 @@ public void onEdgeUnsnap()
 		this.setImage(CLUE_SCROLL_WORLD_IMAGE);
 		this.setImagePoint(CLUE_SCROLL_WORLD_IMAGE_POINT);
 	}
+
+	@Override
+	public void edgeSnapImageHandler(boolean north, boolean south, boolean east, boolean west, BufferedImage image)
```

Comment:
You don't need an entire handler, this could be done much cleaner by changing the default behavior of an edge snapped point to offset draw point such that the entire point image is within the bounds